src/components/Table/TableHeaderColumn.jsx

```diff
@@ -7,12 +7,18 @@ import styles from './Table.css';
 
 export default ({ column, sortClick, sortField, sortState }) => {
   const tooltipId = uuid.v4();
+  const style = {};
+
+  if (column.center) {
+    style.justifyContent = 'center';
+  }
 
   return (
     <MaterialTableHeaderColumn>
       <div
         className={column.sortFn ? styles.headerCell : styles.headerCellNoSort}
         onClick={() => column.sortFn && sortClick(column.field, sortState, column.sortFn)}
+        style={style}
```

Comment:
I know you didn't do this, but you can clean up the className.
```css
.headerCellNoSort {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  text-align: center;
  text-transform: uppercase;
  color: var(--primaryTextColor) !important;
  font-weight: 500;
}

.headerCell {
  composes: headerCellNoSort;
}
```
So we don't need that conditional for headerCell or headerCellNoSort, they are the same. Then, instead of using the style object here, just do a ternary for your justifyContent center.